js/main.js

```diff
@@ -52,7 +52,7 @@ var FilterList = Backbone.Collection.extend({
             error: _.bind(function(model, xhr, options) {
                 //DO NOT LOG OUT IF SERVER ERRORS
```

Comment:
It's been there for a long time...can just leave it for now (we used to log out people if the server errors) in case we decide to go back.